include/cudaq/Optimizer/Dialect/CC/CCOps.td

```diff
@@ -809,6 +808,14 @@ def cc_ExtractValueOp : CCOp<"extract_value", [Pure]> {
   let assemblyFormat = [{
     $container `` $position `:` functional-type(operands, results) attr-dict
   }];
+
+  let builders = [
+    OpBuilder<(ins "mlir::Type":$resultType, "mlir::Value":$aggregate,
+               "std::int64_t":$index), [{
```

Comment:
IIRC the Attribute that we're using here required a signed type. Rather than resist and pollute the code with a bunch of silly casts, I just went with it.

Reply:
Yes. The DenseI64ArrayAttr likes signed better.